### ISSUE TITLE
A zero on its own is translatable content

### ISSUE THREAD

**Issue by johnbillion** [WordPress-Coding-Standards/WordPress-Coding-Standards#1355]
WPCS triggers an i18n error for [this string in core](https://github.com/WordPress/wordpress-develop/blob/a131758bf5dcc420dfed6a832f8e12427220ec78/src/wp-admin/includes/schema.php#L397-L402):

```php
$offset_or_tz = _x( '0', 'default GMT offset or timezone string' );
```

The error is:

```
Strings should have translatable content
(WordPress.WP.I18n.NoEmptyStrings)
```

Should WPCS allow this without triggering an error? Or should we just accept that this is an edge case and ignore it?

**Comment by GaryJones** [comment]
Fixable edge case I'd say.

I suspect [this](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress/Sniffs/WP/I18nSniff.php#L627) could be changed to count the length of the string, or do a strict comparison against `''`.

**Comment by jrfnl** [comment]
PR #1376 fixes this.
